routers/api/v1/repo/release.go

```diff
@@ -147,6 +150,7 @@ func CreateRelease(ctx *context.APIContext, form api.CreateReleaseOption) {
 			IsPrerelease: form.IsPrerelease,
 			IsTag:        false,
 			Repo:         ctx.Repo.Repository,
+			CreatedUnix:  util.TimeStamp(time.Now().Unix()),
```

Comment:
Just for this reason, so caused the issue that repo releases sorted when migrating, `xorm` save current time rather than tag created time. Test passed on local, but CI failed, could you take a look at this?